### DISCUSSION TITLE
PCL2更新建议

### DISCUSSION THREAD

**Discussion by Tski1337** [Meloong-Git/PCL#1886] [讨论]
建议PCL2未来快照版把1.19.2以上版本的自定义皮肤给修复一下！！！
不知道龙猫会不会咕。。。

**Comment by WForst-Breeze** [comment]
可以去 #1596 投票~

**Comment by 20030229** [comment]
能不能加个存档的fabric的更新啊？旧存档需要用的是0.14.10版本的fabric，但是模组需要更高版本，我手动去更新fabric好麻烦，官网下载器什么的我都试了，还是更新不了旧存档的fabric，能不能像hmcl那样能在启动器里更新fabric呢，这对于我这种小白很重要啊的操作

**Reply by Chiloven945** [reply]
有任何提议请单独交一个discussion或者是issue，而不是在跟你说的事情没有任何关系的内容下comment
